src/main/scala/devices/tilelink/Deadlock.scala

```diff
@@ -11,7 +11,8 @@ import freechips.rocketchip.diplomacy._
   *              continue to send traffic to it !!!
   */
 class TLDeadlock(params: DevNullParams, beatBytes: Int = 4)(implicit p: Parameters)
-    extends DevNullDevice(params, beatBytes, new SimpleDevice("deadlock-device", Seq("sifive,deadlock0")))
+    extends DevNullDevice(params, minLatency = 1, // technically not true but we don't want to add extra logic to handle minLatency = 0
+      beatBytes, new SimpleDevice("deadlock-device", Seq("sifive,deadlock0")))
```

Comment:
I'm not sure about this comment. AFAICT, this device never responds. So minLatency=infinity would be more accurate. This device (TLDeadlock) also flagrantly violates the TL forward progress requirements, but I assume based on the name that this is intentional. What is this device for?

Reply:
I don't know I assume you had created it? Maybe @hcook knows?